### PR TITLE
Introduced a master level router for releases

### PR DESCRIPTION
**Pull request by andrewpatto**
Broadly
- removed flowbite react and replaced with daisy ui (WIP)
- introduced breadcrumb pattern and sub pages for Releases (WIP for others like datasets)
- renamed some pages to align language
- introduced visual styling for the difference between activated and inactivated releases
- lots of minor fixes I noticed on the way

### CODE REVIEW

**Comment by andrewpatto** [comment]
Basically there are going to be lots of flaws in this - so we are going to need to revisit in future PRs - rather than keep adding to this one.

For instance - some of the detail detail pages (like Audit Log entries) are disconnected from the breadcrumb structure. But fixing that now would just make this all even bigger.

File: application/frontend/src/pages/releases/detail/releases-detail-sub-page.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      {releaseData.permissionEditSelections && (
+        <FurtherRestrictionsBox
+          releaseId={releaseId}
+          releaseData={releaseData}
+        />
+      )}
+
+      <Box
+        heading="Access Data"
+        applyIsDisabledStyle={!!!releaseData.activation}
```
Comment (DoxasticFox) [inline application/frontend/src/pages/releases/detail/releases-detail-sub-page.tsx]:
```suggestion
        applyIsDisabledStyle={!releaseData.activation}
```

File: application/frontend/src/pages/releases/detail/information-box.tsx [hunk 2]
```diff
@@ -33,54 +33,77 @@ import { useLingui } from "@lingui/react";
...
+    <div className="flex flex-row space-x-4">
+      <button
+        className="btn-success btn-lg btn"
+        disabled={!!releaseData.activation}
+      >
+        Activate Release
+      </button>
+      <button
+        className="btn-warning btn-lg btn"
+        disabled={!!!releaseData.activation}
```
Comment (DoxasticFox) [inline application/frontend/src/pages/releases/detail/information-box.tsx]:
```suggestion
        disabled={!releaseData.activation}
```

Comment (andrewpatto) [reply application/frontend/src/pages/releases/detail/information-box.tsx]: I can keep this going.. `!!!!!`

File: application/frontend/src/pages/releases/detail/information-box.tsx [hunk 3]
```diff
@@ -33,54 +33,77 @@ import { useLingui } from "@lingui/react";
...
+      >
+        Deactivate Release
+      </button>
+    </div>
+  );
 
   return (
-    <Box heading="Release Information">
+    <Box heading={`${releaseData.applicationDacTitle}`}>
```
Comment (DoxasticFox) [inline application/frontend/src/pages/releases/detail/information-box.tsx]: Assuming `releaseData.applicationDacTitle` is a string:
```suggestion
    <Box heading={releaseData.applicationDacTitle}>
```

Comment (andrewpatto) [reply application/frontend/src/pages/releases/detail/information-box.tsx]: Yep - that one had some extras in the heading string that I must have removed

File: application/frontend/src/components/table-footer-paginator.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+/**
+ * A paginator component styled to live in the footer of our paged tables.
+ *
+ * @param props
+ * @constructor
+ */
+export const TableFooterPaginator: React.FC<Props> = (props) => {
+  // for our pagination buttons and text we need to do some page calculations
+  const maxPage = Math.ceil(props.rowCount / props.itemsPerPage);
+  console.log(`Max page = ${maxPage}`);
```
Comment (DoxasticFox) [inline application/frontend/src/components/table-footer-paginator.tsx]: Are these `console.log` calls in here deliberately?

```suggestion
```

Comment (andrewpatto) [reply application/frontend/src/components/table-footer-paginator.tsx]: sorry - yep accidentally left

**Review by DoxasticFox — APPROVED** [review]

**Comment by andrewpatto** [comment]
Thanks Christian!

**Pull request merged**